### PR TITLE
Content/accessibility

### PR DESCRIPTION
**Pull request by accessibilitydesign**

### CODE REVIEW

File: src/accessibility/check-how-accessible-your-content-is/index.md.njk [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+title: Check how accessible your content is
+description: Check how accessible your content is
+section: Accessibility
+theme: Resources
+layout: layout-pane.njk
+order: 4
+---
+If you use Word to write your content you can then run the Flesch reading scale to rate readability. You are aiming for a score of between 70 to 80: 
+
+- 90 to 100 - the document is veryeasy to read
```
Comment (ahosgood) [inline src/accessibility/check-how-accessible-your-content-is/index.md.njk:11]: `veryeasy` -> `very easy`

File: src/accessibility/images/index.md.njk [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+theme: Resources
+layout: layout-pane.njk
+order: 8
+---
+
+Not everyone reading your document will be able to see an image or screenshot so you must always add 'alternative text' or 'alt text'. 
+
+This means people using screen readers can read or hear a description of the image. 
+
+If the image is purely decorative then you can mark it as 'decorative only'. But if this is the case, you might want to think about whether you need to include the image at all. [Images, Describing images, Graphs and charts, Diagrams, Infographics, Image copyright standard](https://www.gov.uk/guidance/content-design/images).
```
Comment (ahosgood) [inline src/accessibility/images/index.md.njk:14]: Might be worth mentioning that for decorative images you should still declare a blank `alt` attribute: `alt=""`

https://www.w3.org/TR/WCAG20-TECHS/H67.html#H67-description

File: src/accessibility/index.md.njk [hunk 3]
```diff
@@ -12,15 +12,16 @@ Every time you send an email, prepare a presentation, write a post or a report y
 
 It is not enough to just make your product look good; it should also work well with assistive technology (AT).  
 
-Your readers may use JAWS, a screen reader or Dragon, a dictation tool. These are [assistive technologies](/accessibility/assistive-technology/)
+Your readers may use JAWS, a screen reader or Dragon, a dictation tool. 
+These are [assistive technologies](/accessibility/assistive-technology/).
```
[the remark this answers is not in the document]
Comment (accessibilitydesign) [reply src/accessibility/index.md.njk]: I already added more info but it's not on your version so I'll push up mine :)

File: src/accessibility/images/index.md.njk [hunk 4]
```diff
@@ -11,6 +11,10 @@ Not everyone reading your document will be able to see an image or screenshot so
...
+If the image is purely decorative then you can mark it as 'decorative only'. 
 
-You can usually add 'alt text' by using your right-hand mouse function on an image, or when you run an accessibility check you'll be guided to add any missing 'alt text'. 
\ No newline at end of file
+But if this is the case, you might want to think about whether you need to include the image at all. [Images, Describing images, Graphs and charts, Diagrams, Infographics, Image copyright standard](https://www.gov.uk/guidance/content-design/images).  
+
+You can usually add 'alt text' by using your right-hand mouse function on an image, or when you run an accessibility check you'll be guided to add any missing 'alt text'. 
+
+If you decide you want to keep the image, [you can mark the image so they can be ignored by assistive technology.[(https://www.w3.org/TR/WCAG20-TECHS/H67.html#H67-description)]
```
Comment (ahosgood) [inline src/accessibility/images/index.md.njk:20]: I think the markup for this is wrong?

Probably should be:

```
If you decide you want to keep the image, you can [mark the image so they can be ignored by assistive technology](https://www.w3.org/TR/WCAG20-TECHS/H67.html#H67-description).
```

**Review by accessibilitydesign — COMMENTED** [review]
already removed the fullstop

**Review by accessibilitydesign — COMMENTED** [review]
all resolved. Will push up

**Review by ahosgood — APPROVED** [review]

**Pull request merged**